src/app/file-manager/file-manager.module.ts

```diff
@@ -28,6 +26,8 @@ import { HTTP_INTERCEPTORS } from '@angular/common/http';
 import { RequestInterceptor } from './services/request.interceptor';
 import { environment } from '@env/environment';
 import { HttpClient } from '@angular/common/http';
+import { FileModalComponent } from './file-modal/file-modal.component';
```

Comment:
Is there a reason why the import paths use `'./file-modal/` instead of `file-modal/`? It's OK either way but I wasn't sure if the usage was intentional or not.

Reply:
Yes, it was intentional because the module specifier need to start with './'. Basically if you're wanting to reference a relative module you must start the module identifier with a dot, so either './' or '../'. Otherwise the module is considered to be a "global" module